engine/src/main/java/org/terasology/rendering/dag/nodes/OpaqueBlocksNode.java

```diff
@@ -122,15 +125,39 @@ public OpaqueBlocksNode(Context context) {
     }
 
     public void enableWireframe() {
+        boolean refreshTaskList = false;
+
+        EnableFaceCulling faceCullingStateChange = new EnableFaceCulling();
+        if (getDesiredStateChanges().contains(faceCullingStateChange)) {
```

Comment:
Because EnableFaceCulling is added as a desired state in the constructor you can rely on the fact that when you enable wireframe it will be there. So, you don't need the IF statement to check for its presence. 

Also notice that in this case the IF statement would always evaluate to false, as you are creating a new EnableFaceCulling state change every time.